### PR TITLE
Slim 873 implement get configuration object request

### PR DESCRIPTION
**Pull request by robinbakkeruscgi**

### CODE REVIEW

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## GetConfigurationObject request
+
+### Description
+GetConfigurationObject is a request to retrieve a ConfigurationObject from on a device. The attributes with OBIS code 0-1:94.31.3.255 is used to access the GPRS_operation_mode setting and following flags:
```
Comment (bvdzwet) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md]: The end of the first sentence looks like the "on" should not be there in "from on a device".

Comment (bvdzwet) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md]: "The attributes with OBIS code ... is" looks not right. It's the configuration object in the electricity meter with the OBIS code that holds a number of attributes as described.

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- HLS_3_on_P3_enable
+- HLS_4_on_P3_enable
+- HLS_5_on_P3_enable
+- HLS_3_on_P0_enable
+- HLS_4_on_P0_enable
+- HLS_5_on_P0_enable
+
+See DSMR document chapter 8.3 for detailed description. The request needs the DeviceIdentification.
+
+All requests have similar response behaviour which is described in [ResponseMessages](./ResponseMessages.md).
```
Comment (bvdzwet) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md]: In US English, "behaviour" is spelled as "behavior".

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObjectResponse.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## GetConfigurationObjectResponse request
+
+### Description
+GetConfigurationObjectResponse returns the result, a ConfigurationObject, which is received from the [GetConfigurationObject](GetConfigurationObject.md) request.
+
+All requests have similar response behaviour which is described in [ResponseMessages](./ResponseMessages.md).
```
Comment (bvdzwet) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObjectResponse.md]: Behavior is without a "u" in US English.

**Comment by bvdzwet** [comment]
When new pages are added like "GetConfigurationObject.md" and "GetConfigurationObjectResponse.md", update "SUMMARY.md" as well to have them included in the index, and to make sure the links in GitBook display a html rendering of the markdown sources.

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md (hunk 2 above)
Comment (robinbakkeruscgi) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md]: done

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md (hunk 1 above)
Comment (robinbakkeruscgi) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md]: done

File: Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObjectResponse.md (hunk 3 above)
Comment (robinbakkeruscgi) [inline Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObjectResponse.md]: done

**Pull request merged**